Description: Compile fix with kernel 4.11
 I use the attached patch to get this module compiled with 4.11. Works
 for me since last sunday.
 Apparently https://aur.archlinux.org/cgit/aur.git/tree/linux411.patch?h=broadcom-wl
 has come with a different patch for the removed last_rx field in
 netdevice. But judging the commit log
 https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/patch/include/linux/netdevice.h?id=4a7c972644c1151f6dd34ff4b5f7eacb239e22ee
 and the fact that last_rx isn't used elsewhere I came up with just
 ifdef it out.
Author: koos vriezen <koos.vriezen@gmail.com>
Bug-Debian: http://bugs.debian.org/861807
Origin: http://bugs.debian.org/861807
Forwarded: no
Reviewed-By: Eduard Bloch <blade@debian.org>
Last-Update: 2017-05-04

--- a/amd64/src/wl/sys/wl_linux.c	2016-12-20 23:11:53.000000000 +0100
+++ b/amd64/src/wl/sys/wl_linux.c	2017-05-01 18:28:01.796506879 +0200
@@ -2912,7 +2912,9 @@ wl_monitor(wl_info_t *wl, wl_rxsts_t *rx
 	if (skb == NULL) return;
 
 	skb->dev = wl->monitor_dev;
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 11, 0)
 	skb->dev->last_rx = jiffies;
+#endif
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 22)
 	skb_reset_mac_header(skb);
 #else
--- a/amd64/src/wl/sys/wl_cfg80211_hybrid.c	2016-12-20 23:11:53.000000000 +0100
+++ b/amd64/src/wl/sys/wl_cfg80211_hybrid.c	2017-05-04 11:02:00.528227486 +0200
@@ -30,6 +30,9 @@
 #include <linux/kthread.h>
 #include <linux/netdevice.h>
 #include <linux/ieee80211.h>
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 11, 0)
+#include <linux/sched/signal.h>
+#endif
 #include <net/cfg80211.h>
 #include <linux/nl80211.h>
 #include <net/rtnetlink.h>
